### PR TITLE
br: set importer branch to release-5.0 for release-5.x versions

### PR DESCRIPTION
**Pull request by glorv**

### CODE REVIEW

**Review by 3pointer — APPROVED** [review]
LGTM

File: jenkins/pipelines/ci/br/br_ghpr_unit_and_integration_test.groovy [hunk 1]
```diff
@@ -82,7 +82,7 @@ def tiflashBranch = TIKV_BRANCH
 def tiflashCommit = ""
 def CDC_BRANCH = ""
 
-if (ghprbTargetBranch == "master")  {
+if (ghprbTargetBranch == "master" || (ghprbTargetBranch.startsWith("release-") && ghprbTargetBranch >= "release-5.0"))  {
```
Comment (kennytm) [inline jenkins/pipelines/ci/br/br_ghpr_unit_and_integration_test.groovy:85]:
```suggestion
if (ghprbTargetBranch =~ /^master$|^release-5\./) {
```

**Review by kennytm — APPROVED** [review]
rest lgtm

**Review by purelind — APPROVED** [review]

**Pull request merged**